### DISCUSSION TITLE
ddospot availability

### DISCUSSION THREAD

**Discussion by iot-operator** [telekom-security/tpotce#904] [General]
The other day I was looking at the docker images and saw that ddospot was not running at all. Checked in a few places, and it seems the compose file is already there, but nothing in /data/ddospot or /opt/ddosbot.
Manually built it and composed.
Just curious, why it is not built and ran automatically?
Is it under development still?
Thanks.

**Comment by t3chn0m4g3** [comment]
You can switch to Nextgen via tped.sh if (run update.sh before that) which will include ddospot. While logging to Elastic works the dashboards still need to be built and some fine-tuning as well.

**Reply by iot-operator** [reply]
Thanks, will do that!

**Comment by t3chn0m4g3** [comment]
I finished the ddospot visualizations. Check the [Wiki](https://github.com/telekom-security/tpotce/wiki) for a brief installation guide. New release is still in development. Stay tuned.
![image](https://user-images.githubusercontent.com/4318452/140557203-078e3e7c-9969-4e3d-82ee-d1ae56e0e115.png)

**Comment by iot-operator** [comment]
Looks awesome, thanks!
